### DISCUSSION TITLE
好きな食べ物を教えて(What is your favorite food?)

### DISCUSSION THREAD

**Discussion by shinkawa-shuan** [NAIST-SE/AttractingContributors#1134] [2024 Attraction Contributions]
Tell me
↓ this
https://github.com/shinkawa-shuan/AttractingContributors-food

**Comment by Drobotsan** [comment]
Hi, I contribute to your project.
Please accept my pull request.

Would you be kind to contribute with my project :D
https://github.com/Drobotsan/Places-in-the-world

Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1140

**Comment by thapakorn604** [comment]
Hi, please accept my request! https://github.com/shinkawa-shuan/AttractingContributors-food/pull/5
btw, if you don't mind please feel free to participate to mine -> https://github.com/NAIST-SE/AttractingContributors/discussions/1115

**Comment by ShunHattori** [comment]
I just made a PR on your repo.
Could you contribute to my repo? Thanks in advance.
[https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git](https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git)
Discussion link: [#1148](https://github.com/NAIST-SE/AttractingContributors/discussions/1148)

**Comment by anzhitingg** [comment]
Hello！
I have just made a PR on your repo. 
Could you contribute to mine? 
Here : https://github.com/anzhitingg/AttractingContributors-MBTI
Thank you very much！

**Comment by r-nishioka** [comment]
I have just made a PR on your repo. 
Could you merge my request and contribute to mine? 
[Where would you like to trip in Japan?](https://github.com/r-nishioka/AttractingContributors-Trip.git)

**Comment by wangzaixiaoliunai** [comment]
I have created a PR.
Could you please contribute to mine?
https://github.com/wangzaixiaoliunai/AttractingContributors-musical_instruments.git

**Comment by Marsall-k** [comment]
I have just made a PR on your Repo!
Could you contribute to mine?
https://github.com/Marsall-k/AttractingContributors-Pokemon

**Comment by Idescrew** [comment]
I have just made a PR on your Repo!
Could you contribute to mine?

https://github.com/Idescrew/AttractingContributors
